File: Partie-Mobile/lib/models/formation.dart
class Formation {
  int? idFormation;
  String? nom;
  String? description;
  String? type;
  String? image;

  Formation(
      {this.idFormation, this.nom, this.description, this.type, this.image});

  Formation.fromJson(Map<String, dynamic> json) {
    idFormation = json['idFormation'];
    nom = json['nom'];
    description = json['description'];
    type = json['type'];
    image = json['image'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data['idFormation'] = this.idFormation;
    data['nom'] = this.nom;
    data['description'] = this.description;
    data['type'] = this.type;
    data['image'] = this.image;
    return data;
  }
}

File: Partie-Backend/src/main/java/com/basketapp/basket_app/models/Formation.java
package com.basketapp.basket_app.models;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.Lob;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Entity
@NoArgsConstructor
@AllArgsConstructor
@Data
public class Formation {

    @Id
    @GeneratedValue (strategy = GenerationType.IDENTITY)
     private  Long idFormation;
     private String nom;
     private String description;
     private String type;
     @Lob
     private String image;
    
}
